addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -574,6 +593,24 @@ var WysiwygMultizone = Wysiwyg.extend({
         this._focusedNode = node;
         this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    _onShowDropdown: function (ev) {
+        ev.preventDefault();
+        $(ev.target).dropdown('show');
+        $(ev.target).dropdown('dispose');
```

Comment:
On show dropdown, show dropdown ?

Tested this, seems to work, just a 2min test though:
```html
<div class="dropdown">
  <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
     Click me
  </button>
  <div class="dropdown-menu">
       <button type="button" class="dropdown-item">
           Item
       </button>
  </div>
</div>
```
```javascript
// Init destroy
$('.dropdown-toggle').removeAttr('data-toggle').dropdown('dispose');

// Own behavior
$('.dropdown-toggle').on('click', function (ev) {
    var $toggle = $(ev.currentTarget);
    $toggle.dropdown($toggle.parent().is('.show') ? 'hide' : 'show');
    $toggle.dropdown('dispose');
});
```

Reply:
Yes but my problem is that I can't catch on click event for .dropdown-toggle and I really don't know why. So, for example, adding an event with something like `'click a.dropdown-toggle': '_onClickDropdowntoggle',` does not work..

Reply:
Probably because bootstrap is removing your handler once it creates the dropdown instance. Try as in my code: Destroy the dropdown on editor start, then attach a click handler "by hand" (no `event` properties).